app/db.py

```diff
@@ -1,8 +1,17 @@
 from sqlalchemy import create_engine, MetaData, Table, Column, Integer, String, DateTime
-# TODO: Add path to table
-engine = create_engine('sqlite://<path_to_DB>', echo = True)
-meta = MetaData()
+from sqlalchemy.ext.declarative import declarative_base
 
-# TODO: Add tables here in PT-5
+# TODO: decide proper place for db
+engine = create_engine('sqlite:///database.db', echo=True)
```

Comment:
The database path will either need to be made relative on both Windows and Linux, docker will need to be used, or a remote database will have to be set up.

Reply:
So it appears that specifically with SQLAlchemy this is a relative path.  This does not need to be changed.